pybot/endpoints/slack/messages.py

```diff
@@ -68,6 +70,24 @@ async def tech_tips(event: Message, app: SirBot):
             logger.debug(f"Exception thrown while logging message_changed {event}")
 
 
+async def search_stackoverflow(event: Message, app: SirBot):
+    """
+    Command to search help links from stackoverflow.
+    """
+    if not_bot_message(event):
+        logger.info(f"tech tips logging: {event}")
+        try:
+            response = await SearchStockOverflow(
+                event["channel"], event["user"], event.get("text"), app
+            )
+            await app.plugins["slack"].api.query(
+                methods.CHAT_POST_MESSAGE, response["message"]
+            )
+
+        except Exception:
+            logger.debug(f"Exception thrown while logging message_changed {event}")
```

Comment:
This is what I see in the logs when I type `!search python`
```
2020-10-13 16:39:33,787 - pybot.endpoints.slack.messages - INFO - tech tips logging: Slack Message: {'client_msg_id': '8022b1f4-feab-4e53-b46e-7b2000c4e9c8', 'type': 'message', 'text': '!search python', 'user': 'U017J0CTMDX', 'ts': '1602625173.002100', 'team': 'T0175BDSM26', 'blocks': [{'type': 'rich_text', 'block_id': '82x', 'elements': [{'type': 'rich_text_section', 'elements': [{'type': 'text', 'text': '!search python'}]}]}], 'channel': 'C017BGVGE4U', 'event_ts': '1602625173.002100', 'channel_type': 'channel'}
2020-10-13 16:39:33,787 - pybot.endpoints.slack.messages - DEBUG - Exception thrown while logging message_changed Slack Message: {'client_msg_id': '8022b1f4-feab-4e53-b46e-7b2000c4e9c8', 'type': 'message', 'text': '!search python', 'user': 'U017J0CTMDX', 'ts': '1602625173.002100', 'team': 'T0175BDSM26', 'blocks': [{'type': 'rich_text', 'block_id': '82x', 'elements': [{'type': 'rich_text_section', 'elements': [{'type': 'text', 'text': '!search python'}]}]}], 'channel': 'C017BGVGE4U', 'event_ts': '1602625173.002100', 'channel_type': 'channel'}
```

It doesn't seem particularly useful for debugging to know the event. We should probably log the error instead of the event if something goes wrong.